guides/setup-dev-environment.md

```diff
@@ -1,96 +1,241 @@
-# Plugin / Dev environment
+# Setup Local Debugging for Spinnaker Services
 
-These notes are very unformatted.
-
-Assume OSX with at least 16GB of memory and 30GB available storage
+## Minimum System requirements
+- Windows or Mac OS X
+- 16GB of Memory
+- 30GB of Available Storage
 
 This allows you to do something like this:
 
-* OSX workstation, with an Ubuntu VM running in multipass, with everything directly wired up.
+* OSX/Windows workstation, with an Ubuntu VM running in multipass, with everything directly wired up.
 * Some services running locally in your workstation (via IntelliJ)
 * All other services running in Minnaker (on the VM)
 
 For example:
-* OSX using IP 192.168.64.1 and the VM using 192.168.64.10
+* OSX/Windows using IP 192.168.64.1 and the VM using 192.168.64.10
 * Orca running on http://192.168.64.1:8083
-* All other services running on 192.168.64.10 (for example, Clouddriver will be on http;//192.168.64.10:7002)
+* All other services running on 192.168.64.6 (for example, Clouddriver will be on http://192.168.64.6:7002)
```

Comment:
i believe this ip address (192.168.64.6) should be 192.168.64.10?

Reply:
@away168 good call out, let me just pick one and stick with it. I ended up have 192.168.64.6 when i ran this, not sure why